src/I18n/En.elm

```diff
@@ -119,10 +119,10 @@ enStrings key =
         -- Index page
         ---
         WelcomeP1 ->
-            "[cCc] TeamWilder is an initiative launched by the Wilidlife Trusts in 2022. (Read more here) The aim of TeeamWilder is to equip everyone with the skills and knowledge they need to make a positive difference for nature - in their own area, and beyond."
```

Comment:
lol Teeeam!

Reply:
I also spotted that. happens to the best of us